docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -0,0 +1,829 @@
+
+:::note
+
+At this time, the Anchor Platform does not send notifications to the wallet application when transaction statuses change, however, it is on our roadmap to add these notifications or "callback requests" so that wallet applications do not have to poll the Anchor Platform for updates.
+
+:::
+
+### Fetching Updates from the Platform
+
+If you only use the Anchor Platform to expose the SEP APIs to wallet applications, then you won't have a strong reason for fetching transaction status updates from the Anchor Platform, mostly because it won't update the transaction status until you make requests to `PATCH /transactions`.
+
+However, if you use the Anchor Platform to monitor the Stellar network for incoming payments (associated with withdrawal transactions), the Anchor Platform will update transaction statuses when payments are received.
+
+There are two ways to fetch updates from the Anchor Platform,
+
+- Polling the Platform API's `GET /transactions/:id` endpoint for the transactions you're expecting a payment for
+- Streaming transaction status change events from a Kafka cluster
+
+While streaming transaction status changes from a Kafka cluster may be a more robust and scalable approach, we're going to use the polling method in this guide. Setting up and using a Kafka cluster will be the subject of a different section of the docs.
+
+:::note
+
+In the future, the Anchor Platform will also support making payments associated with deposit transactions to users on behalf of the business. This is another reason why you may want to use the Anchor Platform's event system.
+
+:::
+
+First, let's configure the Anchor Platform to observe the Stellar network for incoming payments.
+
+<CodeExample>
+
+```yaml
+# docker-compose.yml
+---
+stellar-observer:
+  image: stellar/anchor-platform:edge
+  command: --stellar-observer
+  env_file:
+    - ./dev.env
+  volumes:
+    - ./config:/home
+  depends_on:
+    - db
+```
+
+</CodeExample>
+
+The `--stellar-observer` command starts a process that monitors the distribution accounts configured in your `config.yaml` file as well as those provided as the `destination_account` in `PATCH /transactions` requests for withdrawals.
+
+If a payment is sent to one of these accounts and the memo attached to the transaction matches a `memo` value provided or generated by the Anchor Platform, the Anchor Platform will consider the transaction that memo is associated with as received and update the transaction's status to `pending_anchor`.
+
+Let's make some additions to the `server.js` file so we can poll the Anchor Platform for our expected payments.
```

Comment:
Should it be mentioned that it's possible to use events from Kafka for this purpose?